builtin/user_windows.go

```diff
@@ -194,6 +194,14 @@ type TVINSERTSTRUCT struct {
 
 const nTVINSERTSTRUCT = unsafe.Sizeof(TVINSERTSTRUCT{})
 
+type TVSORT struct {
```

Comment:
Cosmetic, but isn't the structure called TVSORTCB 
I've tried to keep the same names

Reply:
Valid point, I confused myself at one point after I failed to name 
TVSORTCALLBACK > PFNTVCOMPARE 

This then lead me on a rename refactor.

I'll rename this accordingly